tests/network-tests/src/flows/carthage/switchToNPoS.ts

```diff
@@ -0,0 +1,69 @@
+import { extendDebug } from '../../Debugger'
+import { FlowProps } from '../../Flow'
+import { FixtureRunner } from '../../Fixture'
+import { SetForceEraForcingNewFixture } from '../../fixtures/staking/SetForceEraForcingNewFixture'
+import { assert } from 'chai'
+import { ClaimingPayoutStakersSucceedsFixture } from '../../fixtures/staking/ClaimingPayoutStakersSucceedsFixture'
+import { u32 } from '@polkadot/types'
+
+export default async function switchToNPoS({ api, query, env }: FlowProps): Promise<void> {
+  const debug = extendDebug('flow: current era era must be some in NPoS')
+  debug('started')
+  api.enableDebugTxLogs()
+
+  // ----------- ARRANGE ----------------
+
+  // helpers
+  const sleep = (ms: number) => new Promise((resolve) => setTimeout(resolve, ms))
+  const getBalances = async (accounts: string[]) => Promise.all(accounts.map((account) => api.getBalance(account)))
+
+  // constants
+  const sleepTimeSeconds = 10
+
+  const authorities = await api.getSessionAuthorities()
+  const authoritiesStash = (await Promise.all(authorities.map((account) => api.getBonded(account)))).map((x) =>
+    x.unwrap().toString()
+  )
```

Comment:
- `await api.getSessionAuthorities()` already returns the stash account
- `staking.bonded` is a map from stash account to controller account, not the other way around

So I think this should be just:
```suggestion
  const authoritiesStash = await api.getSessionAuthorities()
```